In [2]:
%pylab inline

Populating the interactive namespace from numpy and matplotlib


In [3]:
import matplotlib.pyplot as plt

# Vandermonde Matrix


Talk about what a vandermonde Matrix is and how I created one and the Matrix Random Matrix Sole and vandermonde.cpp

In [9]:
n = open("n.txt").read()
n = n.split("\n")
n.remove("")

residual = open("residual.txt").read()
residual = residual.split("\n")
residual.remove("")

error = open("error.txt").read()
error = error.split("\n")
error.remove("")

print ("n = ", n)
print ("residual = ", residual)
print ("error = ", error)

n =  ['5', '9', '17', '33', '65']
residual =  ['0', '1.08921e-15', '1.55431e-15', '25.394', '50.2465']
error =  ['2.82112e-14', '1.61079e-11', '0.00140697', '3.54709', '5.01578']


As the magnitures of n increase the more ill-conditioned the matrix becomes. This can be seen in the error norm from the beginning of the array to the end. The error norm increases with each increase in n. The residual acts the same way starting at zero and increasing to 50.2465.